<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SupportedFormats</class>
 <widget class="QWidget" name="SupportedFormats">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="soundFormats">
     <property name="title">
      <string>Sound formats</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QRadioButton" name="selectWAV">
        <property name="text">
         <string notr="true">WAV</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="selectMP3">
        <property name="text">
         <string notr="true">MP3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="selectOGG">
        <property name="text">
         <string notr="true">OGG</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="selectFLAC">
        <property name="text">
         <string notr="true">FLAC</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
